src/conditionalAndLoops.js

```diff
@@ -19,160 +19,306 @@
- *  25 to 45 days               |  a month ago
- *  45 to 345 days              |  2 months ago ... 11 months ago
- *  345 to 545 days (1.5 years) |  a year ago
- *  546 days+                   |  2 years ago ... 20 years ago
- * ---------------------------------------------------------------------
- *
- * @param {Date} startDate
- * @param {Date} endDate
- * @return {string}
- *
- * @example
- *   Date('2000-01-01 01:00:00.100'), Date('2000-01-01 01:00:00.200')  => 'a few seconds ago'
- *   Date('2000-01-01 01:00:00.100'), Date('2000-01-01 01:00:05.000')  => '5 minutes ago'
- *   Date('2000-01-01 01:00:00.100'), Date('2000-01-02 03:00:05.000')  => 'a day ago'
- *   Date('2000-01-01 01:00:00.100'), Date('2015-01-02 03:00:05.000')  => '15 years ago'
- *
- */
-function timespanToHumanString(startDate, endDate) {
-	throw new Error("Not implemented");
-}
-
-/**
- * Returns the string with n-ary (binary, ternary, etc, where n<=10) representation of
- * specified number.
- * See more about
- * https://en.wikipedia.org/wiki/Binary_number
- * https://en.wikipedia.org/wiki/Ternary_numeral_system
- * https://en.wikipedia.org/wiki/Radix
- *
- * @param {number} num
- * @param {number} n, radix of the result
- * @return {string}
- *
- * @example:
- *   1024, 2  => '10000000000'
- *   6561, 3  => '100000000'
- *    365, 2  => '101101101'
- *    365, 3  => '111112'
- *    365, 4  => '11231'
- *    365, 10 => '365'
- */
-function toNaryString(num, n) {
-	throw new Error("Not implemented");
-}
-
-module.exports = {
+	if (num % 3 === 0 && num % 5 === 0) {
+	  return "FizzBuzz";
+	} else if (num % 3 === 0) {
+	  return "Fizz";
+	} else if (num % 5 === 0) {
+	  return "Buzz";
+	} else {
+	  return num;
+	}
+  }
+  
+  /**
+   * Returns the factorial of the specified integer n.
+   *
+   * @param {number} n
+   * @return {number}
+   *
+   * @example:
+   *   1  => 1
+   *   5  => 120
+   *   10 => 3628800
+   */
+  function getFactorial(n) {
+	let result = 1;
+	for (let i = 1; i <= n; i++) {
+	  result *= i;
+	}
+	return result;
+  }
+  
+  /**
+   * Returns the sum of integer numbers between n1 and n2 (inclusive).
+   *
+   * @param {number} n1
+   * @param {number} n2
+   * @return {number}
+   *
+   * @example:
+   *   1,2   =>  3  ( = 1+2 )
+   *   5,10  =>  45 ( = 5+6+7+8+9+10 )
+   *   -1,1  =>  0  ( = -1 + 0 + 1 )
+   */
+  function getSumBetweenNumbers(n1, n2) {
+	let result = 0;
+	for (let i = n1; i <= n2; i++) {
+	  result += i;
+	}
+	return result;
+  }
+  
+  /**
+   * Returns true, if a triangle can be built with the specified sides a,b,c and false
+   * in any other ways.
+   *
+   * @param {number} a
+   * @param {number} b
+   * @param {number} c
+   * @return {bool}
+   *
+   * @example:
+   *   1,2,3    =>  false
+   *   3,4,5    =>  true
+   *   10,1,1   =>  false
+   *   10,10,10 =>  true
+   */
+  function isTriangle(a, b, c) {
+	if (a + b > c && b + c > a && c + a > b) {
+	  return true;
+	}
+	return false;
```

Comment:
Can you solve this is just one line? Hint: You just have to return the condition you are checking.